### ISSUE TITLE
Document a way to produce a bundle CSS output

### ISSUE THREAD

**Issue by geelen** [geelen/jspm-loader-css#6]
From https://github.com/geelen/plugin-postcss/issues/5

So I figured out how to do this in my own blog, but it's suuuuper hacky (cause I did at 3am as the "I am gonna crack this before I go to sleep): https://github.com/geelen/glenmaddern.com/blob/master/snapshot.js#L36

I need to figure out a nicer API and show examples. Cause it's possible and cooooool!

**Comment by jgrenat** [comment]
Hi @geelen ,

Is it planned to add support for separateCSS and bundleCSS options from the main css plugin?

**Comment by m4r71n** [comment]
I'm not sure but I think `separateCSS` will also be needed to enable CSP without allowing `unsafe-inline`, isn't it?
Btw.: Thx for your awesome work @guybedford and @geelen!

**Comment by geelen** [comment]
Yeah cool, wiring this in via the options `separateCSS` and `bundleCSS` seem like the smart thing to do.

**Comment by fd** [comment]
@geelen any progress on this?

**Comment by geelen** [comment]
No progress yet. Hoping to be able to jump back on to JSPM stuff in the next week & a bit.

**Comment by MeoMix** [comment]
Any luck with this yet? I'd like to be able to bundle!

**Comment by oligot** [comment]
Just to let you know that I've created a pull request for this issue: #49
